docs/examples/ProgressBar/WithDefaults.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ProgressBar } from '../../../src/index.js';
+
+export default class extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			percentEntered: '',
+			percentCompleteValue: 95,
+		};
+		this.handleChange = this.handleChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	handleChange(event) {
+		this.setState({
+			percentEntered: event.target.value,
+		});
+	}
+
+	handleSubmit(event) {
+		this.setState({
+			percentCompleteValue: this.state.percentEntered,
+			percentEntered: '',
+		});
+		event.preventDefault();
+	}
+
+	render() {
+		return (
+			<div>
+				<form onSubmit={this.handleSubmit}>
+					<input
```

Comment:
Instead of using `input`, let's use a `TextField` instead since that's our version of the input.